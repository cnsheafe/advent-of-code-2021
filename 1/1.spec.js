const path = require("path");
const { countDepthIncreases, countDepthIncreasesFromInput } = require("./1");

describe("day 1", () => {
  test("counts number of times depth has decreased", () => {
    const depthReadings = [199, 200, 208, 210, 200, 207, 240, 269, 260, 263];

    expect(countDepthIncreases(depthReadings)).toBe(7);
  });

  test("count depth decreases from provided input", async () => {
    const filePath = path.resolve("1", "input");
    const count = await countDepthIncreasesFromInput(filePath);

    expect(count).toMatchSnapshot();
  });
});
