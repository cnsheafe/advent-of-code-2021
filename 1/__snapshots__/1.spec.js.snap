// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`day 1 count depth decreases from provided input 1`] = `1342`;
